UPtables/app/controllers/curricula_controller.rb

```diff
@@ -27,7 +27,7 @@ def create
     @curriculum = Curriculum.new(curriculum_params)
 
     respond_to do |format|
-      if @curriculum.save and @curriculum.update_unavailabilities(params[:curriculum_unavailability_ids])
+      if @curriculum.save and @curriculum.update_unavailabilities(params[:curriculum_unavailability_ids]) and @curriculum.assign_ects_modules(params)
```

Comment:
@felixKubicek I think we have to refactor this. If one of the ```#save``` or ```#update``` methods fails, this will trigger a rollback. Here we run all methods non-atomic. We show an error if one method fails, but all preceding actions take place.